File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
function twoSum(nums, target) {
    for(let i = 0; i < nums.length; i++){
        const value = target - nums[i];
        for(let j = 1 + i; j < nums.length; j++){
            if(nums[j] === value){
                return [i,j];
            } 
        }
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
         hashMap = {}

         for i in range(len(nums)):
            hashMap[nums[i]] = i

         for i in range(len(nums)):
            num = target - nums[i]

            if num in hashMap and hashMap[num] != i:
                return [i,hashMap[num]]
         return []